components/mailchimp/mailchimp-form.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import MailchimpInput from './mailchimp-input'
+
+const MailchimpForm = ({ title='', description='', inputFields, listId, buttonLabel, hasShadow }) => (
+	<div id="mc_embed_signup" className={`mailchimp-form ${ hasShadow ? 'shadow' : '' }`}>
```

Comment:
Are all the `mc` and `mce` classes for/from MailChimp's own styling? Because I don't see any styling in this PR.